cmd/kube-controller-manager/app/core.go

```diff
@@ -178,7 +178,7 @@ func startGarbageCollectorController(ctx ControllerContext) (bool, error) {
 	if err != nil {
 		return true, fmt.Errorf("failed to get supported resources from server: %v", err)
 	}
-	deletableResources := discovery.FilteredBy(discovery.SupportsAllVerbs{Verbs: []string{"delete"}}, preferredResources)
+	deletableResources := discovery.FilteredBy(discovery.SupportsAllVerbs{Verbs: []string{"list", "watch", "delete"}}, preferredResources)
```

Comment:
So you want to make sure that for any resource we might want to gc or orphan, that we can get/list/watch/patch/delete?